src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -5190,14 +5190,64 @@ public void CreateTupleTypeSymbol_ElementTypeIsError()
         }
 
         [Fact, WorkItem(13277, "https://github.com/dotnet/roslyn/issues/13277")]
+        [WorkItem(14365, "https://github.com/dotnet/roslyn/issues/14365")]
         public void CreateTupleTypeSymbol_UnderlyingTypeIsError()
```

Comment:
We test that it works in combination with CreateTupleTypeSymbol. Since that was the main motivating factor for adding the new API I don't think it is necessary to create a separate test.